polaris-react/src/components/Inline/Inline.stories.tsx

```diff
@@ -13,95 +13,97 @@ export function Default() {
       <Badge>One</Badge>
       <Badge>Two</Badge>
       <Badge>Three</Badge>
-      <Icon source={CapitalMajor} color="primary" />
+      <Box>
+        <Icon source={CapitalMajor} color="primary" />
+      </Box>
     </Inline>
   );
 }
 
 export function AlignYCenter() {
   return (
     <Inline alignY="center" spacing="1">
+      <Thumbnail source={ImageMajor} alt="example" />
       <Badge>One</Badge>
       <Badge>Two</Badge>
       <Badge>Three</Badge>
-      <Icon source={CapitalMajor} color="primary" />
     </Inline>
   );
 }
 
 export function AlignYTop() {
   return (
-    <Inline alignY="top">
+    <Inline alignY="top" spacing="1'>
+      <Thumbnail source={ImageMajor} alt="example" />
       <Badge>One</Badge>
       <Badge>Two</Badge>
       <Badge>Three</Badge>
-      <Icon source={CapitalMajor} color="primary" />
     </Inline>
   );
 }
 
 export function AlignYBottom() {
   return (
-    <Inline alignY="bottom">
+    <Inline alignY="bottom" spacing="1'>
```

Comment:
```suggestion
    <Inline alignY="bottom" spacing="1">
```

omg I'm sorry, my original suggestion had a syntax error 😨